libs/community/langchain_community/llms/cohere.py

```diff
@@ -88,6 +88,10 @@ class BaseCohere(Serializable):
     streaming: bool = Field(default=False)
     """Whether to stream the results."""
 
+    max_retries: Optional[int] = 3
+    """Maximum number of retries to make when generating."""
+    request_timeout: Optional[float] = None
```

Comment:
Out of curiosity, is there a reason why this is defaulted to `None` instead of `60` as in Cohere's API ([ref](https://github.com/cohere-ai/cohere-python/blob/8a63bca5b15c2f1298d57094ab956a8315bd990e/src/cohere/client.py#L67))?

```suggestion
    request_timeout: Optional[float] = 60
```